conjure-go-client/httpclient/internal/errors.go

```diff
@@ -32,3 +32,17 @@ func StatusCodeFromError(err error) (statusCode int, ok bool) {
 	statusCode, ok = statusCodeI.(int)
 	return statusCode, ok
 }
+
+// LocationFromError retrieves the 'location' parameter from the provided werror.
+// If the error is not a werror or does not have the location param, ok is false.
+//
+// The default client error decoder sets the location parameter on its returned errors
+// if the status code is 3xx and a location is set in the response header
+func LocationFromError(err error) (location string, ok bool) {
```

Comment:
good point. this actually helps found a bug in the code, same for `StatusFromError` too

Reply:
so the second return value from ParamFromError indicates the param is safe or not, rather than exist or not.
we need to check if the returned value is nil or not.

Reply:
awesome, glad it helped find uncover issues!!